### ISSUE TITLE
Права доступа

### ISSUE THREAD

**Issue by asnenne** [instantsoft/icms2#128]
Было бы не плохо, если бы свойствам категорий можно было установить права для чтения и изменения, как это реализовано при создании полей.

**Comment by iAndroS** [comment]
Согласен. Вообще, свойства категорий должны наследовать практически весь функционал полей.

**Comment by iAndroS** [comment]
Подумал тут на сон грядущий: а зачем, собсна, движку нужны свойства категорий? Почему изначально поля типов контента не сделали по аналогии с виджетом - для родительской категории минимально обязательный набор, а для вложенных уже индивидуальные поля? А в списке полей категории под названием поля выводить список категорий, где еще применяется данное поле с теми же настройками... 
Я вспомнил тут просто: когда я только начинал свое знакомство, я так и думал, что в настройке типа контента я указываю общие для всех категорий поля, а потом уже буду добавлять для каждой категории этого типа контента свои поля. Я еще в ступор впал, не найдя такой функции. Пока разобрался, что это свойства, времени потратил немало. 
Приведу в пример один скриншот, вводящий в ступор меня, как админа
![screenshot_1](https://cloud.githubusercontent.com/assets/17408903/13726904/6f866068-e916-11e5-9203-19c6737a112b.png)
Не очень-то удобно...

**Comment by fuzegit** [comment]
Из скриншота видно лишь то, что у вас много одинаковых свойств зачем то.

**Comment by iAndroS** [comment]
Есть разные категории товаров, к примеру, и в каждой категории товаров в свойствах грубо говоря  "Вид товара" означает подкатегорию товаров, ибо нет смысла плодить лишние уровни вложенности

**Comment by fuzegit** [comment]
Так вы сами себя запутали, создав много свойств с одинаковым названием, вместо того, чтобы использовать одно, прикрепляя его к нужным категориям.

**Comment by iAndroS** [comment]
там по-другому никак. Это вообще на основе авипарса от локанафт. Можете уточнить его точку зрения, может, он удачнее пояснит...
